lib/kit.js

```diff
@@ -55,7 +58,7 @@ class CoinsenceKit {
             })
           if (!deployEvent) { throw new Error('No DeployInstance event found') }
 
-          return Promise.resolve({ txHash: txHash, daoAddress: deployEvent.args.dao, apps: apps});
+          saveDao(spaceId, JSON.stringify({ txHash: txHash, daoAddress: deployEvent.args.dao, apps: apps}));
```

Comment:
I would not do that in here but in the http app. 
We should simply return the transaction here. then wherever this library is used can decide on what to do here.